recipe/meta.yaml

```diff
@@ -57,7 +58,6 @@ outputs:
         - six
       run:
         - {{ pin_compatible('numpy', lower_bound='1.16') }}
-        - arrow-cpp {{ version }}
```

Comment:
Just as a note, `arrow-cpp` is [using `run_exports`]( https://github.com/conda-forge/arrow-cpp-feedstock/blob/7f706f6416776024420c271bc8bdebce54ae7801/recipe/meta.yaml#L17-L18 ), which is why we are able to drop this. It was added before because relying on the build string for CUDA selection seems to mess up the `run_exports` mechanism.

Reply:
Had to restore `arrow-cpp` in `run`.